specification/cat/nodeattrs/CatNodeAttributesRequest.ts

```diff
@@ -28,4 +28,15 @@ import { CatRequestBase } from '@cat/_types/CatBase'
  * @doc_id cat-nodeattrs
  * @cluster_privileges monitor
  */
-export interface Request extends CatRequestBase {}
+export interface Request extends CatRequestBase {
```

Comment:
To reduce code duplication, would it help to add a `CatWithLocalRequestBase` that each of these can extend?

Reply:
sure that works too, I'll do it

Reply:
This requires a change in the compiler: https://github.com/elastic/elasticsearch-specification/blob/48e2d9de9de2911b8cb1cf715e4bc0a2b1f4b827/compiler/src/model/build-model.ts#L347-L349

Reply:
don't think it's worth modifying the compiler for cat, so if it's okay I'd merge it as is